src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
```

Comment:
Avoid using literals in expressions (HARDCODE). Instead, use constants to explain these values. Declare a constant for 3 and give it a meaningful name, like `SHORT_TERM_RENTAL`.